README.md

```diff
@@ -128,6 +128,49 @@ You can then view route data for each service:
     linkerd routes authors
     ```
 
+---
+
+## Traffic Splits
+
+You can use a modified version of booksapp to demo a trafficsplit.
+
+1. Install `booksapp-trafficsplit.yml` which includes the MySQL backend, a
```

Comment:
i noticed that `replicas` is set to `1` in `booksapp-trafficsplit.yml` rather than the `3` in `mysql-app.yml`. intentional?

Reply:
Yes, good eye. I did reduce the number of replicas to reduce complexity, and should mention this in the updated docs. However, is there a benefit to keeping the number of replicas 3?

Reply:
I would guess with 45% confidence that setting replicas > 1 demonstrates that this is a truly distributed app backed by a stateful mysql db. Also maybe it shows off load balancing and makes our stats look more interesting? I am betting @klingerf has more context.

Reply:
Thanks. As I think about it, it may make more sense to leave the `replicas` and the `FAILURE_RATE` as they are in the original booksapp, to be consistent.

Reply:
I restored the original `replicas` number but left out the `FAILURE_RATE` as it's simpler when testing trafficsplit metrics to start out expecting 100% success and tweak if needed.